tldw_Server_API/app/api/v1/API_Deps/DB_Deps.py

```
@@ -108,13 +112,12 @@ def _get_db_path_for_user(user_id: int) -> Path:
         logger.error(f"Could not resolve database directory for user_id {user_id}: {e}", exc_info=True)
         raise OSError(f"Could not initialize storage directory for user {user_id}.") from e
 
-def _resolve_media_db_for_user(current_user: User) -> MediaDatabase:
+def _get_or_create_media_db_factory(current_user: User) -> MediaDbFactory:
```

Ruff (0.15.6)

[warning] Too many branches (16 > 12)

(PLR0912)